composer.json

```diff
@@ -18,16 +18,16 @@
         "exclude-from-classmap": [ "/Tests/" ]
     },
     "require": {
-        "php": "^7.1",
-        "imagine/imagine": "^1.1",
+        "php": "^7.1|^8.0",
+        "imagine/imagine": "^1.2.4",
```

Comment:
As noted in the commit message:

Older versions of imagine/imagine had incompatibilities with PHP 8, even though their composer.json stated it supported PHP 8.

See https://github.com/avalanche123/Imagine/pull/740/commits/e68b92b3e9abf3d50c4e94ab9e5e46a0c84a2179.